tests/prepare/basic/data/main.fmf

```diff
@@ -3,7 +3,7 @@ provision:
     image: fedora
 prepare:
     how: install
-    package: tmt
+    package: fmf
 execute:
     how: tmt
-    script: tmt --help
+    script: fmf --help
```

Comment:
Do these changes look unrelated, or am I missing something?

Reply:
Without this change this test will keep failing until fixed tmt rpm reaches stable repositories.
IMO fmf is good enough to test that package installation works (or any other package for that manner)

Reply:
Ah, we assert the output as well, need to amend that as well.

Reply:
Fixed in a14e15fe

Reply:
Understood.